### PR TITLE
feat (fast-form-generator): update pop-up menu style, address incorrect shadowing 

### PR DESCRIPTION
**Pull request by nalogiudice**
Closes [#666](https://github.com/Microsoft/fast-dna/issues/666)

<body>

<footer>
 

For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

File: packages/fast-development-site-react/src/components/toc/toc-item.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ const style: ComponentStyles<ITocItemManagedClasses, IDevSiteDesignSystem> = {
         color: (config: IDevSiteDesignSystem): string => {
             return config.foregroundColor;
         },
-        boxShadow: `0 ${toPx(2)} ${toPx(4)} ${toPx(-1)} #ccc`,
+        boxShadow: `0 ${toPx(2)} ${toPx(4)} rgba(0, 0, 0, 0.06)`,
```
Comment (janechu) [inline packages/fast-development-site-react/src/components/toc/toc-item.tsx]: It seems that we still have the issue that the drop shadow is visible over the canvas. What was this change meant to fix?

File: packages/fast-form-generator-react/src/utilities/form-input.style.ts [hunk 2]
```diff
@@ -270,7 +270,7 @@ export function applyPopupMenuStyles(): ICSSRules<{}> {
         width: toPx(200),
         background: colors.menuGray,
         zIndex: "1",
-        top: toPx(55),
+        top: toPx(78),
```
Comment (janechu) [inline packages/fast-form-generator-react/src/utilities/form-input.style.ts]: this looks wrong to me.
screen shot 2018-07-11 at 4 40 30 pm

Comment (nalogiudice) [reply packages/fast-form-generator-react/src/utilities/form-input.style.ts]: screen shot 2018-07-11 at 7 23 03 pm
Looks fine to me? Where are you seeing this?

File: packages/fast-development-site-react/src/components/toc/toc-item.tsx (hunk 1 above)
Comment (nalogiudice) [reply packages/fast-development-site-react/src/components/toc/toc-item.tsx]: The notes said to update to current design, so I simply used the box shadow data found in the design on Figma. This is how it looks now:
screen shot 2018-07-11 at 9 01 19 pm
Is this the expected rendering? Seems to match the Figma spec.

Comment (janechu) [reply packages/fast-development-site-react/src/components/toc/toc-item.tsx]: Figma file does not have this shadow overlapping the canvas: 
screen shot 2018-07-12 at 9 07 38 am

**Pull request merged**